[TransformStream article] Use new tag and aside (#8279)
GoogleChrome/web.dev: src/site/content/en/blog/transformstream/index.md

author: thomassteiner
tags:
  - blog
  - newly-interoperable
---

{% Aside 'celebration' %}

> Woohoo, this web feature is now usable on all browsers.

Reviewer: ```suggestion
This web feature is now available in all three engines!
```

Reviewer: Try to avoid "all browsers" as frequently that is not the case, even when it is available in all engines.
